plugins/polio/api/vaccines/stock_management.py

```diff
@@ -263,7 +275,7 @@ def get_list_of_unusable_vials(self):
                 or report.stock_correction == IncidentReport.StockCorrectionChoices.VACCINE_EXPIRED
                 or report.stock_correction == IncidentReport.StockCorrectionChoices.VVM_REACHED_DISCARD_POINT
                 or report.stock_correction == IncidentReport.StockCorrectionChoices.UNREADABLE_LABEL
-                or report.stock_correction == IncidentReport.StockCorrectionChoices.OTHER
+                # or report.stock_correction == IncidentReport.StockCorrectionChoices.OTHER
```

Comment:
`OTHER` looks commented everywhere. Is this OK?

Reply:
Maybe add a comment about that so that a new reader will understand why this is commented?

Reply:
It think this will be updated in the base branch